app/services/transaction_search/searcher.rb

```diff
@@ -44,6 +61,12 @@ def options
         @search_options.dup
       end
 
+      def to_h
```

Comment:
This seems like a weird method name to me. Not what I really think `to_h` does...

Reply:
It is converting itself to a hash... Might be doing too much to be named that. Maybe `to_options` or `to_options_by_searcher`? This is in a `Results` object so it's a hash keyed by the searcher's key and the values are the array of options (e.g. facilities, products) to be used on the resulting form.

Reply:
I think I like `to_options_by_searcher`. That seems much clearer IMO.